app/models/spree_avatax_certified/response.rb

```diff
@@ -0,0 +1,32 @@
+module SpreeAvataxCertified
+  class Response
+    def initialize(response)
+      @response = response
+    end
+
+    def success?
+      return false if error?
+      @response['ResultCode'] == 'Success' || @response[:result_code] == 'Success'
+    end
+
+    def error?
+      @response == 'error in Tax' || @response['ResultCode'] == 'Error' || @response[:result_code] == 'Error'
```

Comment:
It's inappropriate to have an object contain either a random string `error in Tax`, or a hash.

You need to set-up proper exception handling. Easiest way if you don't need specific error messages would be to have response be `nil`.

Again, those strings 'error in Tax` appear very random.